epub33/core/index.html

```diff
@@ -7850,144 +7850,182 @@ <h4>Content document dimensions</h4>
+						aria-details="flow-scrolled-continuous-diagram"
+						alt="The progression of a continuous scroll of content extends vertically off the user's screen,
+						with new documents added to the bottom as encountered." />
+				</figure>
+
+				<details id="flow-scrolled-continuous-diagram" class="desc">
+					<summary>Image description</summary>
+					<p>A single, column-like strip (i.e., a rectangle without a bottom edge) with a text flowing
+						down the strip. The text is sectioned with headers figuring 'Chapter 1', '2'. The top part
+						of the strip is enclosed in a schematic view of a tablet.</p>
+				</details>
+
+				<figure id="fig-fxl-flow-scrolled-continuous">
+					<figcaption>Rendering of a fixed layout EPUB publication with multiple spine items, and with the <code>rendition:flow</code> set to <code>scrolled-continuous</code>.</figcaption>
+					<img src="images/example_rendering_FXL_scrolled_continuous.svg" width="220"
+						aria-details="flow-fxl-scrolled-continuous"
+						alt="The progression of a continuous scroll of content extends vertically off the user's screen, with new documents following without gaps." />
+				</figure>
+
+				<details id="flow-fxl-scrolled-continuous" class="desc">
+					<summary>Image description</summary>
+					<p>A single, column-like strip (i.e. a rectangle without a bottom edge) with multiple comic pages flowing down the strip. The content is segmented by the page boundaries, but there are no gaps between the pages. The top part of the strip is enclosed in a schematic view of a tablet.</p>
+				</details>
+
+				<figure id="fig-flow-scrolled-doc">
+					<figcaption>Rendering of an EPUB publication with multiple spine items, and with the
+						<code>rendition:flow</code> set to <code>scrolled-doc</code>.</figcaption>
+					<img src="images/example_rendering_scrolled_doc.svg" width="600"
+						aria-details="flow-scrolled-doc-diagram"
+						alt="The progression of scrollable documents depicting how only the content within each document
+						is scrollable." />
+				</figure>
+
+				<details id="flow-scrolled-doc-diagram" class="desc">
+					<summary>Image description</summary>
+					<p>Three column-like strips (i.e., a rectangles without bottom edges) linked left-to-middle and
+						middle-to-right with respective arrows, each containing a text flowing down the strip. The
+						text is sectioned with headers figuring 'Chapter 1', '2' and '3'. Each strip starts with a
+						chapter header and flows down the strip. The top part of the leftmost strip is enclosed in a
+						schematic view of a tablet.</p>
+				</details>
+
+				<figure id="fig-fxl-flow-scrolled-doc">
+					<figcaption>Rendering of a fixed layout EPUB publication with multiple spine items, and with the <code>rendition:flow</code> set to <code>scrolled-doc</code>.</figcaption>
+					<img src="images/example_rendering_FXL_scrolled_doc.svg" width="600" 
```

Comment:
```suggestion
					<img src="images/example_rendering_FXL_scrolled_doc.svg" width="900" 
```

To make the size of the figure consistent with the others.